client/lib/wp/sync-handler/index.js

```diff
@@ -159,38 +206,20 @@ export class SyncHandler {
 				}
 			};
 
+			const checkPagination = res => {
+				if ( hasPaginationChanged( res, localResponseBody ) ) {
+					// clearPageResultsForSite( reqParams )
+				}
```

Comment:
It appears that `cacheResponse` resolved with the serverResponse, so this should return `res` if we want to add more handlers in the future.